include/circt/Dialect/Calyx/CalyxStructure.td

```diff
@@ -104,6 +104,15 @@ def ComponentOp : CalyxOp<"component", [
       /// Returns the body of a Calyx component.
       Block *getBody() { return &getOperation()->getRegion(0).front(); }
 
+      /// Returns the port information for this component.
+      SmallVector<PortInfo> getPortInfo();
```

Comment:
Sorry for not noticing this earlier; should be unneeded since CellInterface::portInfos returns the same thing.